app/assets/stylesheets/application.scss

```diff
@@ -63,3 +63,22 @@
 .gem-c-radio__input {
   font-size: inherit;
 }
+
+// overwrites
+
+// `content-block` leaks styles into the `govuk-tabs` component so we need to
+// overwrite a few attributes to defend the component and preserve these values
+.content-block {
+  .govuk-tabs {
+    ul {
+      padding: 0;
+      margin: 0;
+    }
+
+    li {
+      list-style: none;
```

Comment:
I think this might have affected the content on certain pages.

For example https://www.gov.uk/view-driving-licence#other-ways-to-apply on the section under the `## Apply by post` there is a list `<li>` that isn't showing the bullet points because of this style (found by @keithiopia).